File: BergerMsfaApi/ClientApp/src/app/Shared/Entity/JourneyPlan/JourneyPlan.ts
import { NgbDate } from '@ng-bootstrap/ng-bootstrap';
import { ModeOfApprovals } from '../../Enums/modeOfApproval';

export class JourneyPlan {

    public id: number;
    public dealers: number[];
    //public employeeRegId: string;
    //public approvedBy?: number;
    public visitDDate: NgbDate;
    public visitDate: string;
    //public approvedDate: string;
    //public approvedStatus: boolean;

    constructor() {
        this.id = 0;
        this.dealers=[]
        //this.code = 0,
        //this.employeeRegId = '';
        //this.approvedBy = null
        //this.approvedDate = '';
        //this.approvedStatus = false;
     
    }
   
}

File: Berger.Data/MsfaEntity/JourneyPlan/JourneyPlan.cs
using Berger.Data.Common;
using System;

namespace Berger.Data.MsfaEntity
{
    public class JourneyPlan: AuditableEntity<int>
    {
        public string Code { get; set; }
        public int EmployeeRegId { get; set; }

        public DateTime VisitDate { get; set; }
        public int? ApprovedBy { get; set; }
        public DateTime? ApprovedDate { get; set; }
    }


}
